.github/actions/pr-review/pr_review.py

```
@@ -0,0 +1,1057 @@
+    return payload
+
+
+def provider_configuration() -> tuple[str, str]:
+    litellm_url = os.environ.get("LITELLM_BASE_URL", "")
+    litellm_key = os.environ.get("LITELLM_API_KEY", "")
+    openai_key = os.environ.get("OPENAI_API_KEY", "")
+    if litellm_url and litellm_key:
+        return litellm_url.rstrip("/") + "/chat/completions", litellm_key
+    if openai_key:
+        return "https://api.openai.com/v1/chat/completions", openai_key
+    raise ModelOutputError("no LLM credential was supplied")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)